governance.md

```diff
@@ -0,0 +1,340 @@
+Main NumFOCUS Board. While the Steering Council and Main NumFOCUS Board will
+work together closely on the BDFL selection process, the Main NUMFOCUS Board
+will make the final decision.
+
+
+Steering Council
+----------------
+
+The Project will have a Steering Council that consists of Project Contributors
+who have produced contributions that are substantial in quality and quantity,
+and sustained over at least one year. The overall role of the Council is to
+ensure, through working with the BDFL and taking input from the Community, the
+long-term well-being of the project, both technically and as a community.
+
+During the everyday project activities, council members participate in all
+discussions, code review and other project activities as peers with all other
+Contributors and the Community. In these everyday activities, Council Members
+do not have any special power or privilege through their membership on the
+Council. However, it is expected that because of the quality and quantity of
+their contributions and their expert knowledge of the Project Software and
+Services that Council Members will provide useful guidance, both technical and
+in terms of project direction, to potentially less experienced contributors.
+
+The Steering Council and its Members play a special role in certain situations.
+In particular, the Council may:
+
+-   Make decisions about the overall scope, vision and direction of the
+    project.
+-   Make decisions about strategic collaborations with other organizations or
+    individuals.
+-   Make decisions about specific technical issues, features, bugs and pull
+    requests. They are the primary mechanism of guiding the code review process
+    and merging pull requests.
+-   Make decisions about the Services that are run by The Project and manage
+    those Services for the benefit of the Project and Community.
+-   Make decisions when regular community discussion doesn’t produce consensus
+    on an issue in a reasonable time frame.
+
+### Council membership
+
+To become eligible for being a Steering Council Member an individual should
+
+-  Be a Project Contributor who has produced contributions that are substantial in
+quality and quantity
+-  Sustain this development consistently over several months
+-  Support the community through code review, answering user questions,
+   triaging bug reports, participating constructively in broader conversations,
+   and being willing to help with documentation and infrastructure.
+-  Demonstrate breadth by supporting the community outside of their particular
+   work interests or sub-project
+-  Be civil in public discourse
+
+Potential Council Members are nominated by existing Council members and voted
+upon by the existing Council after asking if the potential Member is interested
+and willing to serve in that capacity. The Council will be initially formed
+from the set of existing Core Developers who, as of 2019, have ownership rights
+over the organization.
```

Comment:
Could we list the acting steering council members here?

Reply:
If so then we'll have to keep this up to date.  I would only want to do this if someone volunteers to keep this maintained for the next year or two.

Reply:
Given that no one volunteered to do this I'm going to suggest that we not do this.

Reply:
Thus far it's not clear to me how much this list is expected to change over the next year or two. This is both important for determining the risk of it going stale as well as the amount of work any one individual will take on. Any thoughts?

Reply:
Do we want a one to one relationship between GitHub owners in the steering Council?

If not, we’ll be updating this as soon as the next member is added. We can move to an explicit list at that time?

Reply:
I had assumed that the steering council were also owners (and vice versa), but maybe that is not a reasonable assumption. How do you see these differing?

Reply:
I've gone ahead and pushed a change stating there is a 1-to-1 relationship of steering council members and owners. Please let me know your thoughts on this in a new diff for clarity (if that's alright).